### PR TITLE
Change status check to include 4 (finished) and 5 (seeding)

### PR DESCRIPTION
**Pull request by pseudo-rnd-thoughts**
https://libtorrent.org/reference-Torrent_Status.html#torrent_status

### CODE REVIEW

File: src/AutoROM.py [hunk 1]
```diff
@@ -153,32 +164,32 @@ def torrent_tar_to_buffer():
...
+
+        # libtorrent params
+        ses = lt.session()
+        params = lt.parse_magnet_uri(uri)
+        params.save_path = save_path
+        handle = ses.add_torrent(params)
+
+        # download roms as long as state is not seeding
+        timeit = 1
+        while handle.status().state != 5:
```
Comment (jjshoots) [inline src/AutoROM.py:179]: Do we want to check for either state 4 and 5 instead of only 5? Just a thought

File: src/AutoROM.py [hunk 2]
```diff
@@ -153,32 +164,32 @@ def torrent_tar_to_buffer():
...
-
-                if timeit >= 20:
-                    raise TimeoutError("Took too long to download and reach seeding.")
-
-            success = True
-
-        except TimeoutError:
-            pass
-
+            raise RuntimeError("Tried to download ROMs 3 time, which have all failed, please try again or report this issue.")
```
Comment (jjshoots) [inline src/AutoROM.py:167]: 3 times*, small typo

**Review by jjshoots — CHANGES_REQUESTED** [review]
LGTM, two small comments

File: src/AutoROM.py (hunk 1 above)
Comment (pseudo-rnd-thoughts) [reply src/AutoROM.py:179]: I was unsure, I had it in the original PR. I will add now and see what happens for users

**Pull request merged**